Tests/Common/IntegrationTestFixtureBase.Asserts.cs

```diff
@@ -103,16 +103,10 @@ public async Task AssertNetworkServerModuleLogStartsWithAsync(string logMessageS
                 new SearchLogOptions(string.Concat(logMessageStart1, " or ", logMessageStart2)));
         }
 
-        public async Task<SearchLogResult> SearchNetworkServerModuleAsync(Func<string, bool> predicate, SearchLogOptions options = null)
-        {
-            SearchLogResult searchResult;
-            if (this.udpLogListener != null)
-                searchResult = await SearchUdpLogs(predicate, options);
-            else
-                searchResult = await SearchIoTHubLogs(predicate, options);
-
-            return searchResult;
-        }
+        public async Task<SearchLogResult> SearchNetworkServerModuleAsync(Func<string, bool> predicate, SearchLogOptions options = null) =>
+            this.tcpLogListener != null
+                ? await SearchUdpLogs(predicate, options)
+                : await SearchIoTHubLogs(predicate, options);
```

Comment:
Technically, we should have removed IoT Hub log searching in #788, but removing it and making TCP logging optional doesn't make sense at all because then none of the tests would pass. This is something I thought could be revisited more properly as part of #740 when we want to add back IoT Hub logging.

/cc @Mandur

Reply:
I'm confused. We remove the IoT Hub logger here, because we say it's not used, but then we talk about bringing it back as a logging provider in #740 ?

Reply:
We remove it from the static logger, but logging to iot hub is a desired feature which is desirable, therefore we will need it as part of the ILogger. make sense?

Reply:
> I'm confused. We remove the IoT Hub logger here, because we say it's not used, but then we talk about bringing it back as a logging provider in #740 ?

So one of the story's acceptance criteria was “Logging sinks are migrated”. Since #740 was going to revisit the IoT Hub logging, it was decided to migrate the minimum set of sinks that were necessary for now and on which the tests depend the most.

Reply:
I think we want to keep feature parity with what we had previously. We do not know, what existing users of the kit actually use, hence we need to get IoT hub logging back in using ILogger sinks.

Reply:
Since #740 is tracking this, I resolve this comment.